sagemaker-pyspark-sdk/src/sagemaker_pyspark/algorithms/LinearLearnerSageMakerEstimator.py

```diff
@@ -516,13 +516,6 @@ class LinearLearnerBinaryClassifier(SageMakerEstimatorBase, LinearLearnerParams)
                              "recall_at_target_precision. Must be in range [0, 1]. ",
                              typeConverter=TypeConverters.toFloat)
 
-    positive_example_weight_mult = Param(Params._dummy(), "positive_example_weight_mult",
```

Comment:
this is an unrelated change to the readme. Not sure if it was intentional?

Reply:
Yep this is a typo we made before release. We removed setter and getter of this hyper-parameter but forgot to remove this declaration. This hyper-parameter is unimplemented by algorithm team. So setting it doesn't make any difference(but doesn't cause trouble either, just ignored by the image).